data-prepper-plugins/aws-sqs-common/src/main/java/org/opensearch/dataprepper/plugins/aws/sqs/common/model/SqsOptions.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.aws.sqs.common.model;
+
+import java.time.Duration;
+
+public class SqsOptions {
+    private static final int DEFAULT_MAXIMUM_MESSAGES = 10;
+    private static final Duration DEFAULT_POLL_DELAY_SECONDS = Duration.ofSeconds(0);
```

Comment:
Use `null` as the default. This will use the SQS queue defaults. See my comment on maximum messages.